python/numpy-in-pytorch-modules.yml

```diff
@@ -0,0 +1,10 @@
+rules:
+- id: numpy-in-pytorch-modules
+  patterns:
+    - pattern: $RESULT = numpy.$FUNCTION(...)
+    - pattern-inside: |
+        class $MODULE(torch.nn.Module):
```

Comment:
Will it work if `Module` is imported directly or if it is aliased to sth? We need testcases for this

Reply:
It works! I'll add a test.